src/transformStateWithClones.js

```diff
@@ -7,7 +7,46 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentState = simpleObjectClone(state);
```

Comment:
```suggestion
  let currentState = { ...state };
```
we can do it simpler